cloud/scope/machine.go

```diff
@@ -130,9 +130,37 @@ func (m *MachineScope) DiskSpecs() []azure.DiskSpec {
 	spec := azure.DiskSpec{
 		Name: azure.GenerateOSDiskName(m.Name()),
 	}
+
 	return []azure.DiskSpec{spec}
 }
 
+// SubnetSpecs return the subnet specs.
+func (m *MachineScope) SubnetSpecs() []azure.SubnetSpec {
```

Comment:
you shouldn't need this, MachineScope doesn't need to implement SubnetScope because AzureMachine doesn't reconcile subnets

Reply:
removed